packages/xo-server/src/xo-mixins/backups-ng-logs.js

```diff
@@ -1,8 +1,14 @@
+import asyncMap from '@xen-orchestra/async-map'
+import createLogger from '@xen-orchestra/log'
 import ms from 'ms'
 import { forEach, isEmpty, iteratee, sortedIndexBy } from 'lodash'
 
 import { debounceWithKey } from '../_pDebounceWithKey'
 
+const consoleLogger = createLogger('xo:xo-mixins:backups-ng-logs')
```

Comment:
Why `console`?

Reply:
Because `logger` was used to store logs. I updated it in the bellow commit.